core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/QcType.java

```diff
@@ -246,4 +247,14 @@ public String toString() {
     sb.append(getUnits());
     return sb.toString();
   }
+
+  @Override
+  public long getId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getQcTypeId" on line 72. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)